### ISSUE TITLE
No documentation what so ever?

### ISSUE THREAD

**Issue by Jimmy-Z** [lkl/linux#481]
I have so many questions.

> SYSCTL
key: "sysctl" value type: string
Configure sysctl values of the booted kernel via the hijack library. Multiple entries can be specified.

How? separated by new line? space? colon? semi-colon?

> Interface type
key: "type" value type: string
The interface type in host operating system to connect to LKL. The following example specifies a tap interface.

OK I get it, "tap" means tap, but what else do you support?

> Interface parameter
key: "param" value type: string
Additional configuration parameters for the interface specified by Interface type (type). The parameters depend on the interface type.

In summary, there is basically no information in those sentences.

Why is there a "global" gateway AND per interface gateway?

The example json config only shows how the network is configured, what about file system? does the hijack mechanism supports only network calls? if not, how is that configured?
